builtin/logical/pki/cert_util.go

```diff
@@ -150,6 +151,10 @@ func (sc *storageContext) fetchCAInfoByIssuerId(issuerId issuerID, usage issuerU
 	return caInfo, nil
 }
 
+func fetchCertBySerialBigInt(ctx context.Context, b *backend, req *logical.Request, prefix string, serial *big.Int) (*logical.StorageEntry, error) {
+	return fetchCertBySerial(ctx, b, req, prefix, certutil.GetHexFormatted(serial.Bytes(), ":"))
```

Comment:
nit: wanna use the normalization call?

Reply:
Sure I'll add a new `serialFromBigInt` to the `utils.go` and refactor `serialFromCert` to use the new method as well.

Reply:
Done.